File: Leetcode Challenge/March/Java/Smallest String With A Given Numeric Value.java
class Solution {
    public String getSmallestString(int n, int k) {
        char res[] = new char[n];
        Arrays.fill(res, 'a');
        k -= n;
        
        while(k > 0){
            res[n-1] += Math.min(25, k);
            k -= Math.min(25, k);
            n--;
        }
        return String.valueOf(res);
    }
}

File: Leetcode Challenge/March/C++/Smallest String With A Given Numeric Value.cpp
class Solution {
public:
    string getSmallestString(int n, int k) {
        string res(n,'a');
        k -= n;
        
        while(k > 0){
            res[n-1] += min(25, k);
            k -= min(25, k);
            n--;
        }
        return res;
    }
};
